ci/tests/general/general_vdisk.py

```diff
@@ -128,7 +128,7 @@ def delete_volume(vdisk, vpool, loop_device=None, root_client=None, wait=True):
                 raise RuntimeError('Disk {0} was not deleted from model after {1} seconds'.format(volume_name, timeout))
 
     @staticmethod
-    def write_to_volume(vdisk=None, vpool=None, location=None, count=1024, bs='1M', input_type='random', root_client=None):
+    def write_to_volume(vdisk=None, vpool=None, location=None, count=1024, bs='1M', input_type='random', root_client=None, is_symlink=False):
```

Comment:
I wouldn't add yet another parameter
With root_client.file_is_link you can check whether a file is a link and make this parameter obsolete